src/test/kotlin/dummyData/DummyRecipes.kt

```diff
@@ -1227,7 +1227,58 @@ object DummyRecipes {
                 "Serve in warm tortillas with salsa."
             ),
             description = "Spicy potato and chorizo street tacos.",
-            ingredients = listOf("potatoes", "chorizo", "corn tortillas", "salsa", "cilantro")
+            ingredients = listOf("Potatoes", "chorizo", "corn tortillas", "salsa", "cilantro")
+        )
+    )
+
+    val nonPotatoRecipes = listOf(
+        createDummyRecipe(
+            name = "Spaghetti Carbonara",
+            id = "PASTA_001",
+            minutes = 25,
+            contributorId = "user_italian",
+            submitted = LocalDate.parse("2023-03-15"),
+            tags = listOf("pasta", "italian", "main dish"),
+            nutrition = Nutrition(
+                calories = 450.0,
+                fat = 18.0,
+                sugar = 3.0,
+                sodium = 600.0,
+                protein = 20.0,
+                saturatedFat = 7.0,
+                carbohydrates = 50.0
+            ),
+            steps = listOf(
+                "Cook pasta until al dente.",
+                "Mix with egg, cheese, and pancetta.",
+                "Serve hot with black pepper."
+            ),
+            description = "Classic Italian pasta with creamy sauce.",
```

Comment:
I believe you only need to pass the ingredients to `createDummyRecipe` since your logic depends on them.